Sidney-/slugtrade
# login extends base and logout redirects to home page

Changes:
- The login page now extends the base page
- Logout takes you back to the home page instead of a dedicated logout page

How to test:
- Make sure that the login screen now also shows contents of base.html
- Make sure that logout redirects to home page and logs the user out
